test/config/karma.unit.js

```diff
@@ -7,7 +7,7 @@ var singleStart = process.env.SINGLE_START;
 var coverage = process.env.COVERAGE;
 
 // configures browsers to run test against
-// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'IE', 'PhantomJS' ]
+// any of [ 'ChromeHeadless', 'Chrome', 'Firefox' ]
```

Comment:
As we now test against Safari, we could add it here as well

```suggestion
// any of [ 'ChromeHeadless', 'Chrome', 'Firefox', 'Safari' ]
```

Reply:
Already mentioning in the [contributing guide](https://github.com/bpmn-io/bpmn-js/commit/5680826e9e272da3713811e738738b3a14afe479).